tools/performance-server/src/main/kotlin/routes/route.kt

```diff
@@ -64,6 +64,22 @@ object LocalCache {
     fun buildExists(target: String, buildNumber: String) =
             buildsInfo[target][buildNumber]?.let { true } ?: false
 
+    fun delete(target: String, builds: Iterable<String>, bintrayUser: String, bintrayPassword: String) {
```

Comment:
Why `delete` is needed at all?

Reply:
We have to remove some builds after some time. We couldn't save all builds for very long period. Moreover, in case of some runs with errors.